README.md

```diff
@@ -219,26 +219,43 @@ seiso images history "$OPENSHIFT_PROJECT/$APP_NAME" --force
 
 ## Development
 
-### Requirements
+Requirements:
 
 * go 1.13
 * goreleaser
 * Docker
 
-### Build
+Run code during development:
+```console
+go run main.go
+```
+
+Reformat all Go code:
+```console
+make fmt
+```
```

Comment:
This seems unnecessary. Why not just do `go fmt` which is literally the same as `make fmt`, but without the `find . -type f -name '*.go' -exec go fmt {} \;`, as `go fmt` without arguments does just that?

Reply:
Unfortunately, it's not. `go fmt` requires a (list of) Go file(s) as an argument, which all have to be in the same directory. Wrapping all commands in the `Makefile` is meant to make things easier.

Of course, you can still run the `go` commands separately for the file you're working on.

Reply:
True `go fmt` only does it for current workdir, but then `go fmt ./...` does it recursively.

Reply:
Nice! I'll fix that.